frontend/src/component/pages/shared/CreateShortLinkSection.tsx

```diff
@@ -59,6 +61,12 @@ export class CreateShortLinkSection extends Component<Props> {
           </div>
         </div>
         <div className={'input-error'}>{this.props.inputErr}</div>
+        <div>
+          <Toggle onClick={this.props.onPublicToggle}>
+            Make this short link public?
+            <br />(&nbsp;will be publicly listed&nbsp;)
+          </Toggle>
```

Comment:
Few questions here:
- I wonder whether we can separate toggle description from the `Toggle` button itself so that we can leave developers flexibility for customization
- Instead of having extract description inside the parentheses,  can we make the description explicit to users? For example, change to 
```html
<!-- 
Margin right on desktop, full width on mobile. 
public-listing-toggle has border or background color.
-->
<div class="public-listing-toggle">Shared on <a>public feed</a>
    <Toggle onClick={this.props.onPublicToggle}></Toggle>
</div>
``` 
<img width="1069" alt="Screen Shot 2020-04-03 at 8 24 09 PM" src="https://user-images.githubusercontent.com/3537801/78417664-3da0ce80-75e9-11ea-80f6-4fb072873396.png">

The screenshot is just an example.

Reply:
I like both those ideas, will make the toggle more clear and concise and the component itself will be more flexible. 👍

Reply:
TODO
- [ ] Change label to "Shared on <a>public feed</a>"
- [ ] Separate label subcomponent from `Toggle` component, have the `Toggle` be a child of it
- [ ] Change style of the label subcomponent, play with border and background color